src/context/GenderContext.js

```diff
@@ -1,15 +1,16 @@
-import React, { useState, createContext } from 'react';
+import React, { createContext } from 'react';
 import PropTypes from 'prop-types';
 
-import { MALE } from '../constants/genders';
+import { MALE, GENDER } from '../constants/genders';
```

Comment:
Instead of GENDER just declare local const in this module, for example: `const GENDER_LOCAL_STORAGE_KEY = 'DAILY_ROUTINE_GENDER'`.